syntax = "proto3";

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}

service Compute {
    rpc compute (ComputeRequest) returns (ComputeReply) {}
}
message ComputeRequest {
    string Reports = 1;
    string Codes = 2;
    string FixedFiles = 3;

}
message ComputeReply {
    string RecommendFiles = 1;
    string Metric = 2;
}

//python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. ./RPC.proto